modules/aerospike/aerospike.go

```diff
@@ -0,0 +1,71 @@
+package aerospike
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	port          = "3000/tcp"
+	containerName = "tc_dynamodb_local"
```

Comment:
bug: let's remove this const, as it's not used. OTOH we discourage the usage of fixed container names, unless used for Reusing containers.
```suggestion
	containerName = "tc_dynamodb_local"
```